### PR TITLE
fix broken links in quickstarts

### PR DESCRIPTION
**Pull request by polandll**
[Daniel Santos] in slack
Hey folks, I think this broken link should be replaced with a more stable version such as
[https://stargate.io/docs/latest/develop/attachments/restv2.html]https://stargate.io/docs/latest/develop/attachments/restv2.html

### CODE REVIEW

File: docs-src/stargate-develop/modules/quickstart/pages/qs-document.adoc [hunk 1]
```diff
@@ -56,5 +56,5 @@ include::develop:page$api-doc/doc-document-deleting.adoc[Deleting documents and
 
 Voila! For more information on the Document API, see the
 xref:develop:dev-with-doc.adoc[Using the Document API]
-or the https://stargate.io/docs/stargate/1.0/attachments/docv2.html[Document API] in the
+or the https://stargate.io/docs/latest/develop/attachments/docv2.html[Document API] in the
```
Comment (msmygit) [inline docs-src/stargate-develop/modules/quickstart/pages/qs-document.adoc:59]: @polandll how about making it like `xref:develop:api.adoc[Document API]` so we make folks to the API landing page where they come to know about other APIs too? Thoughts?

File: docs-src/stargate-develop/modules/quickstart/pages/qs-rest.adoc [hunk 2]
```diff
@@ -50,5 +50,5 @@ include::develop:page$api-rest/rest-deleting-rows.adoc[Deleting rows in your tab
 
 Voila! For more information on the REST API, see the see the
 xref:develop:dev-with-rest.adoc[Using the REST API]
-or the https://stargate.io/docs/stargate/1.0/attachments/restv2.html[REST API] in the
+or the https://stargate.io/docs/latest/develop/attachments/restv2.html[REST API] in the
```
Comment (msmygit) [inline docs-src/stargate-develop/modules/quickstart/pages/qs-rest.adoc:53]: @polandll how about making it like `xref:develop:api.adoc[REST API]` so we make folks to the API landing page where they come to know about other APIs too? Thoughts?

**Review by msmygit — APPROVED** [review]
LGTM. I added a trivial comment about cross-refencing to the other api doc.

File: docs-src/stargate-develop/modules/quickstart/pages/qs-document.adoc (hunk 1 above)
Comment (polandll) [reply docs-src/stargate-develop/modules/quickstart/pages/qs-document.adoc:59]: Nah - it's easy to find the API references - it's right there in the lefthand nav. Let's just pop them here to the exact API.

**Pull request merged**